src/App.js

```diff
@@ -6,15 +6,26 @@ import Navbar from 'react-bootstrap/Navbar'
 import Nav from 'react-bootstrap/Nav'
 import NavDropdown from  'react-bootstrap/NavDropdown'
 import {Component} from "react";
+import { BrowserRouter, Route, Switch} from "react-router-dom";
+import Menu from "./components/Menu";
+import Map from './components/Map';
+import Home from './components/Home';
 
 class App extends Component {
   render() {
-      return (
-        <div className="App">
-            <Header />
-        </div>
-      );
-    }
+    return (
+      <div className="App">
+        <Header />
+        <BrowserRouter>
+          <Switch>
+            <Route path="/menu/:restaurant" component={Menu} />
```

Comment:
Will we end up hard-coding the different restaurant possibilities or will we source this from some sort of API?

Reply:
this is hardcoded date in restaurants.data.js. i'd prefer to hardcode data in this case coz we control the json format it's just for the usability test

Reply:
Why remove the Header? I think I've fixed it on my branch.

Reply:
just added back, so on my pr, header will overshadow the other components. but once your commits are in, it will be good.